### ISSUE TITLE
A11y_AzureToolsStorageExplorer_UpdateAccessTier_ScreenReader: NVDA not announces "Learn More" link in descriptive.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#1920]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #MAS2.4.4; #NVDA; #JAWS; #Win32; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Right click on blob and select "Change Access Tier" control and hit enter.
7. "Update Access Tier" popup gets opened.
8. Start NVDA.
9. Navigate using Tab key to "Learn More" link, verify NVDA not announces "Learn More" in descriptive way.

**​Actual:​**
While navigating using Tab key to "Learn More" link, NVDA not announces "Learn More" link in descriptive way. Instead it announces other content present on the screen with "Learn More" link.

Same issue is observed in "**JAWS**" also.

**Expected:**
While navigating using Tab key to "Learn More" link, NVDA should announce "Learn More" link in more descriptive way.

**UserImpact:**
Screen reader users face difficulty to know the purpose of the link if screen reader not announces link in more descriptive way.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

**Attachment for Reference:**
[A11y_AzureToolsStorageExplorer_UpdateAccessTier_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3656436/A11y_AzureToolsStorageExplorer_UpdateAccessTier_ScreenReader.pptx)

**Comment by craxal** [comment]
@Suraj1200499 NVDA does announce the learn more link, although it's hard to catch when it reads the rest of the contents of the dialog. So we still need to fix that somehow...

I've noticed that some dialogs read only the first control when tabbed to as expected, but others read off the entire contents. Still investigating what may be the difference. Attempting to programmatically focus onto the first element doesn't fix anything.

**Comment by craxal** [comment]
@Suraj1200499 NVDA has an option **Browse Mode** > **Automatic Say All on page load**. If this option is not checked, the entire contents of the dialog should not be read, and you should hear the screen reader announce the link.

**Comment by Suraj1200499** [comment]
@Craxal Facing issue in opening the application and will check on other system and will update you by tomorrow.

**Comment by MRayermannMSFT** [comment]
This is not something we'd put in the VPAT. This behavior is the result of a NVDA specific behavior.

**Comment by craxal** [comment]
Independent of window behavior, the link itself does not provide enough context, so we should change the link text to something like "Learn more about hot, cool, and archive access tiers...".

**Comment by JasonYeMSFT** [comment]
Related to #1908. Since this is a design choice made by NVDA, we are not going to do anything to prevent NVDA from reading the entire page.

**Comment by craxal** [comment]
@JasonYeMSFT Please refer to my previous comment.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
